### PR TITLE
eslint support

### PR DESCRIPTION
**Pull request by aoushana**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (briangonzalez) [inline .eslintrc]: @aoushana Can we make sure this file uses 2 spaces? I see 4 in some spots.

**Comment by briangonzalez** [comment]
Nice work @aoushana 😄  

Have you been able to test it after your changes?

**Comment by aoushana** [comment]
@briangonzalez Thanks! My code was broken at first because I was joining the color array by an empty string. It works again.

**Comment by briangonzalez** [comment]
@aoushana Can you rebase (_see_ squash) this work into a single commit?

**Comment by aoushana** [comment]
@briangonzalez I learned about rebasing and squashed my 6 commits into a single commit. Theres a few issues I couldn't solve. I hope we can look at that together.

**Pull request closed without merging**